shared/protocol/src/main/java/io/pravega/shared/protocol/netty/ReplyProcessor.java

```diff
@@ -61,6 +61,8 @@ default void process(Reply reply) {
     void segmentCreated(WireCommands.SegmentCreated segmentCreated);
 
     void segmentsMerged(WireCommands.SegmentsMerged segmentsMerged);
+    
+    void segmentsMergedBatch(WireCommands.SegmentsBatchMerged segmentsMerged);
```

Comment:
The method name should match the wire command name.

Reply:
done